### DISCUSSION TITLE
ressample all

### DISCUSSION THREAD

**Discussion by flvribeiro** [highfestiva/finplot#136] [Q&A] [answered]
is there any way to resample all the data?

I use the same graph to plot data from several different timeframes and tickers, such as indiex and assets, but the scales + indicators break when the time series type changes.

I sove this with the fplt.close () command, but it excludes the chart's structure and the transition is terrible.

so i wondered if you thought of any easy way to redefine the axes.

Beautiful work! Thanks...

**Comment by highfestiva** [comment, marked as answer]
Sounds like you want to do something similar to [BeeSee20Eye35](https://github.com/highfestiva/finplot/issues/95#issuecomment-753408373).

**Reply by flvribeiro** [reply]
yesss!

exactly loss of scale and data after the update. I will try to update the internal df as mentioned.

"plots[0].datasrc.df =..."

tks!

**Comment by flvribeiro** [comment]
Perfect!!

            self.ax.vb.datasrc.df = self.ax.vb.datasrc.df[0:0]
            self.plots[0].ax.vb.datasrc.df = self.plots[1].ax.vb.datasrc.df = self.plots[2].ax.vb.datasrc.df = self.ax.vb.datasrc.df

called when the tmf or symbol changes

tks!!
